### DISCUSSION TITLE
Radial neighbor list length of type 1 is not enough

### DISCUSSION THREAD

**Discussion by GitWhilebear** [deepmodeling/deepmd-kit#123] [Q&A] [answered]
hi~
when I try to train a Al2O3 model with my Al2O3_se_r.json and you Al2O3 data sets, I get this Warning. So I want to know how to set the parameter sel, why should this parameter be set and what does this parameter do in descriptor ?

Al2O3_se_r.json:
{
    "_comment": " model parameters",
    "model_type":	"se_r",
    "model":	{
	"type_map":		["Al", "O"],
	"descriptor": {
	    "type":		"se_r",
	    "sel":		[68, 102],
	    "rcut_smth":	4.00,
	    "rcut":		8.00,
	    "neuron":		[5, 10, 20],
	    "resnet_dt":	false,
	    "seed":		1,
	    "_comment": " that's all"
	},
	"fitting_net" :{
	    "neuron":		[120, 120, 120],
	    "resnet_dt":	true,
	    "seed":		1,
	    "_comment":		"that's all"
	},
	"_comment":	" that's all"
    },

    "learning_rate" : {
	"start_lr":	0.005,
	"decay_steps":	5000,
	"decay_rate":	0.95,
	"_comment":	" that's all"
    },

    "loss" : {
	"start_pref_e":	0.02,
	"limit_pref_e":	1,
	"start_pref_f":	1000,
	"limit_pref_f":	1,
	"start_pref_v":	0,
	"limit_pref_v":	0,
	"_comment":	" that's all"
    },

    "_comment": " traing controls",
    "training" : {
	"systems":	["/data/deepmd-se-dataset/Al2O3"],
	"set_prefix":	"set",    
	"stop_batch":	5000000,
	"batch_size":	1,

	"seed":		1,

	"_comment": " display and restart",
	"_comment": " frequencies counted in batch",
	"disp_file":	"lcurve_Al2O3_ser_01.out",
	"disp_freq":	100,
	"numb_test":	10,
	"save_freq":	1000,
	"save_ckpt":	"Al2O3_ser_01.ckpt",
	"load_ckpt":	"Al2O3_ser_01.ckpt",
	"disp_training":true,
	"time_training":true,
	"profiling":	false,
	"profiling_file":	"timeline.json",
	"_comment":	"that's all"
    },

**Comment by njzjz** [comment, marked as answer]
Same as #108.
